test/test-services.env

```diff
@@ -0,0 +1,8 @@
+RUN_REDIS_TESTS=1
+OPENTELEMETRY_REDIS_HOST=localhost
+OPENTELEMETRY_REDIS_PORT=63790
+
+RUN_MONGODB_TESTS: 1
```

Comment:
you started with `=` then changed to `:`, is that on purpose? can we keep it consistent?

similar to a previous comment, just a preference, but I think using `true` instead of `1` is more clear, because that number makes me think the test will run 1 time, but if I change to 2 it would run twice

Reply:
> you started with `=` then changed to `:`

Oh, wow. I hadn't realized I did that. I'm kinda shocked that that works.  That feels like `dotenv` having [DWIM'd](https://en.wikipedia.org/wiki/DWIM) it.

> makes me think the test will run 1 time, but if I change to 2 it would run twice

Ha. Good argument for `true`.

Reply:
> you started with `=` then changed to `:`

That works with dotenv:

```
> dotenv.parse(fs.readFileSync('test/test-services.env'))
{
  RUN_REDIS_TESTS: '1',
  OPENTELEMETRY_REDIS_HOST: 'localhost',
  OPENTELEMETRY_REDIS_PORT: '63790',
  RUN_MONGODB_TESTS: '1',
  MONGODB_DB: 'opentelemetry-tests',
  MONGODB_HOST: '127.0.0.1',
  MONGODB_PORT: '27017'
}
```

but not with Node recent `--env-file` FWIW:

```
% node --env-file ./test/test-services.env
Welcome to Node.js v22.5.1.
Type ".help" for more information.
> process.env.RUN_REDIS_TESTS
'1'
> process.env.MONGODB_HOST
undefined
```

Reply:
commit 503f867c

Reply:
TIL about the dotenv 😄